### PR TITLE
#157639065 Completed Sever and base app set up

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR do?
Sets up server and base app
#### Description of Task to be completed?
Setup server, setup server directories (Server/), install express, body-parser, morgan, errorhandler, babel and nodemon as a dev dependency. Create app.js as the entry point to the server.
#### How should this be manually tested?
Using Postman send a GET to 'baseurl/'
#### What are the relevant pivotal tracker stories?
#157639065
#### Screenshots (if appropriate)

screenshot 122
screenshot 123

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env", "es2015"]
```
Comment (paradisekelechi) [inline .babelrc]: Why are you using "env" and "es2015" together?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "babel-cli": "^6.26.0",
+        "babel-preset-env": "^1.7.0",
+        "babel-preset-es2015": "^6.24.1",
+        "babel-register": "^6.26.0",
+        "body-parser": "^1.18.3",
+        "errorhandler": "^1.5.0",
+        "express": "^4.16.3",
+        "morgan": "^1.9.0"
+    },
+    "devDependencies": {
```
Comment (paradisekelechi) [inline package.json:29]: I cannot find eslint here. Are you using eslint?

Comment (chukwuemekachm) [reply package.json:29]: I am yet to set up eslint

File: app.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+/** route handlers */
+app.get('/', (req, res) => res.status(200).json({ message: 'Welcome to Maintenance Tracker' }));
+
+app.all('*', (req, res) => res.status(404).json({ message: 'Route not supported on server.' }));
+
+/** bootups */
+app.listen(port, () => console.log(`Server listening on port ${port}...`));
+
+/** export */
```
Comment (paradisekelechi) [inline app.js]: This comment is unnecessary

File: app.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/** imports */
+import express from 'express';
+import logger from 'morgan';
+import errorhandler from 'errorhandler';
+import bodyParser from 'body-parser';
+
+/** instantiations */
```
Comment (paradisekelechi) [inline app.js]: Not necessary

**Comment by chukwuemekachm** [comment]
I will fix that

**Pull request merged**